docs/Using-Pantheon/Truffle.md

```diff
@@ -0,0 +1,59 @@
+description: Using Pantheon with Truffle
+<!--- END of page meta data -->
+
+# Using Pantheon with Truffle 
+
+Developing for Pantheon using Truffle is the same as using Truffle to develop for the public Ethereum networks. 
+Truffle supports Pantheon with the only difference being Pantheon does not implement private key management. 
+To use Pantheon with Truffle, you must configure the Truffle wallet.
```

Comment:
I'm not sure if we should call it "THE Truffle wallet" or "A Truffle wallet". I believe there are other wallet implementations for truffle out there.

Reply:
But this seems to be the "official" implementation so it should be fine.

Reply:
I would go with "a Truffle wallet" - there are a few

Reply:
Done